src/redis-benchmark.c

```diff
@@ -1617,7 +1617,14 @@ int main(int argc, const char **argv) {
     argc -= i;
     argv += i;
 
+    slot_placeholder = zmalloc(1);
```

Comment:
Why not use `sdsnew` instead?

Reply:
swapped to static string pointer as per Oran's review